osf/migrations/0197_draft_node_permissions.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-07-14 19:04
+from __future__ import unicode_literals
+
+import logging
+from django.db import migrations
+from django.core.management.sql import emit_post_migrate_signal
+
+logger = logging.getLogger(__name__)
+
+
+def post_migrate_signal(state, schema):
+    # this is to make sure that the draft registration permissions created earlier exist!
+    emit_post_migrate_signal(3, False, 'default')
+
+
+add_draft_read_write_admin_auth_groups = """
+    INSERT INTO auth_group (name)
```

Comment:
one more thought: is it feasible to have tests for this migration? we should probably avoid raw, untested SQL to the extent we reasonably can.

Reply:
Added tests in the latest commit (see `osf_tests/test_migration_sql.py`)